### PR TITLE
feat/SA-251: upgrading to latest notifications module

### PR DESCRIPTION
**Pull request by wozzer72**
As a result of SA-251 (teams) and SA-258 (posts enrichment) upgrading to use the latest notifications modules.

Includes mapping of account ids to name - anomaly is deployed into root account (mgmt) and removing the defunct slack channel and username.

Includes upgrading the terraform lock file while here; both for latest modules and multi-platforms.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -3,14 +3,18 @@
...
 
   allowed_aws_services = ["budgets.amazonaws.com", "costalerts.amazonaws.com", "lambda.amazonaws.com"]
   create_sns_topic     = local.enable_sns_topic_creation
   email                = local.email
+  enable_slack         = local.enable_slack
   slack                = local.slack
   sns_topic_name       = var.sns_topic_name
   tags                 = var.tags
+  accounts_id_to_name = {
+    "536471746696" = "root"
```
Comment (gambol99) [inline main.tf:16]: Should this be here? … needs to be passed as a variable no ..

Comment (wozzer72) [reply main.tf:16]: Yes. This is the top level module that instantiates the instance of notifications - thus it's the one that knows where it is going to run. The cost-anomaly runs in the root account.

Comment (wozzer72) [reply main.tf:16]: Oh crap. That's not true is it. Just checked the actions, and there is no deployment of terraform. I'll update this once I've found which repo is dependent on this module.

Comment (wozzer72) [reply main.tf]: Updated this now.

**Review by gambol99 — APPROVED** [review]
LGTM

**Pull request merged**